### DISCUSSION TITLE
Folder.jpg as album art and volume button question

### DISCUSSION THREAD

**Discussion by Stealth3** [TT-ReBORN/Georgia-ReBORN#45] [Q&A] [answered]
How do I use folder.jpg as the album art for a folder? I think it's currently getting the art from somewhere else because I'm playing an album now and it's not showing any art despite having the cover as folder.jpg in the album folder. Also its not showing lyrics because it doesn't have any album art?

Also is there a way to always show the volume bar instead of having to click the volume button to show it first?

**Comment by TT-ReBORN** [comment]
If you have folder.jpg/png or cover.jpg/png in your root album folder it should always display the correct album art.
Try other albums and see if it works. For more information about folder structure check "How do you display the disc art in Details?"
here: https://github.com/TT-ReBORN/Georgia-ReBORN#faq

If there is no album art, there will be automatically a music note place holder displayed. Also it should display lyrics if the server found some, otherwise there will be a message displayed no lyrics found. I have previously mentioned users should tinker settings in the Lyric Show 3 settings in Preferences, maybe some setting messed something up...

The volume button bar is currently always hidden, I can make an option to always display it.

**Reply by Stealth3** [reply]
Here is a screenshot. It has Folder.jpg in the folder and displays in other players. [Screenshot](https://i.postimg.cc/tJy6TPx8/SS-1.jpg)

**Reply by TT-ReBORN** [reply]
Really strange, does this happen only with this specific album and other albums working fine?
It's also really strange that you don't have the music note placeholder if no album art is found...

**Reply by TT-ReBORN** [reply]
One other thing, can you show me the folder structure of this album and can you open and display the folder.jpg?
Just to be sure the jpg is not somehow corrupted.

**Reply by Stealth3** [reply]
[Folder / Image Open](https://i.postimg.cc/GmcHyd9T/SS-2.jpg) also displays in Musicbee and GoneMade (for Android)

**Reply by TT-ReBORN** [reply]
Does this happen ONLY with this album or other albums as well?
Can you contact me please on HydrogenAudio for further instructions?
https://hydrogenaud.io/index.php?action=profile;u=139848

**Reply by Stealth3** [reply]
So far only this album (I think), which is what lead me to believe it was not using Folder.jpg and was pulling the artwork from a database.

**Reply by Stealth3** [reply]
I messaged you on hydro

**Reply by TT-ReBORN** [reply]
Are you using the original Georgia-ReBORN config with its components from the zip or did you install additional user-components and changed some settings?

**Reply by Stealth3** [reply]
All I did was install it from the zip from this github and play like 5 songs before coming here to ask some config questions. I only installed the audioscrobbler plugin.

**Comment by TT-ReBORN** [comment, marked as answer]
Case closed, the uploader renamed a WebP image to jpg and got marked as corrupt, this image spread around the web =)
Source of the album art cover is from Discogs: https://img.discogs.com/uhrQTNaOKnC53GBZycO4IR1Lmhc=/fit-in/600x600/filters:strip_icc():format(webp):mode_rgb():quality(90)/discogs-images/R-2263438-1273118187.jpeg.jpg

**Comment by TT-ReBORN** [comment]
@Stealth3

Btw I added now the option to permanently show the volume bar.
It can be found in top menu Options > Player controls > Show volume control button > Auto-hide bar

Download as usual from [here](https://github.com/TT-ReBORN/Georgia-ReBORN/archive/refs/heads/master.zip) and replace your existing files.

-TT
